app/config/config.yml.dist

```diff
@@ -240,6 +247,9 @@ wysiwyg:
 #  encryption: null
 #  auth_mode: null
 
+#mailoptions:
+#  transport: mail
+
```

Comment:
My immediate feeling is one of unease at having this in two places, people could uncomment *both* and then not understand the reason that `mail()` isn't firing.